src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMappingTest.java

```diff
@@ -233,6 +261,24 @@ void testBuildHearingWindowNullReturn() {
         assertThat(result.getDateRangeEnd()).isNull();
     }
 
+    @DisplayName("When case when not autolist and an urgent case, buildHearingWindow returns start date of tomorrow")
+    @Test
+    void testBuildHearingWindowNotAutoListIsUrgent() {
+        SscsCaseData caseData = SscsCaseData.builder()
+                .dwpResponseDate("2021-12-01")
+                .urgentCase("Yes")
+                .build();
+        HearingWindow result = HearingsDetailsMapping.buildHearingWindow(caseData, false);
+
+        assertThat(result).isNotNull();
+
+        LocalDate expected = LocalDate.now().plusDays(DAYS_TO_ADD_HEARING_WINDOW_TODAY);
+        assertThat(result.getDateRangeStart()).isEqualTo("2021-12-15");
```

Comment:
Minor: expected isn't used here?

Reply:
Agreed and worth removing this to avoid future confusion.  Good spot Fergal ;)